crates/astria-conductor/src/bin/astria-conductor.rs

```diff
@@ -35,25 +27,24 @@ use tracing::{
     instrument,
 };
 
+const EXIT_CONFIG_CODE: i32 = 78;
```

Comment:
Can you link the systemd man page for this and maybe call it `EX_CONFIG` so other folks immediately know where this comes from?

We should probably also create a tracking issue to address this in all our services and implement this somehow: https://doc.rust-lang.org/std/process/trait.Termination.html